package.json

```diff
@@ -23,15 +23,19 @@
         "express-jwt": "^5.3.1",
         "express-jwt-authz": "^2.3.1",
         "express-openapi-validator": "^1.6.2",
+        "google-libphonenumber": "^3.2.6",
         "got": "^9.6.0",
-        "helmet": "^3.15.0",
+        "helmet": "^3.21.2",
         "json-pointer": "^0.6.0",
+        "libphonenumber-js": "^1.7.30",
```

Comment:
Is this and google-libphonenumber required?

Reply:
removed